app/templates/_package.json

```diff
@@ -20,7 +20,8 @@
     "bluebird": "^2.9.34",
     "connect-mongo": "^0.8.1",<% } %><% if (filters.sequelize) { %>
     "sequelize": "^3.5.1",
-    "sqlite3": "~3.0.2",<% } %><% if (filters.auth) { %>
+    "sqlite3": "~3.0.2",
+    "express-sequelize-session": "0.4.0",<% } %><% if (filters.auth) { %>
```

Comment:
We really only need `connect-mongo` or `express-sequelize-session` when `filters.twitterAuth`. Would you mind adding the `if (filters.twitterAuth)` for those two deps?